python/cudnn/moe_ep/_megamoe.py

```
@@ -0,0 +1,485 @@
+            w2,
+            qcfg=qcfg,
+            comm="torch_dist",
+            bwd_impl="mega" if op.generate_c else "replay",
+        )
+
+    # -- layout adapters --------------------------------------------------
+
+    def _to_kernel_weights(self, fc1_weight: MoeTensor, fc2_weight: MoeTensor):
+        """(E,H,2I) gate-first / (E,I,H)  ->  w13 (E,2I,H) up-first / w2 (E,H,I)."""
+
+        I = self._op.intermediate_size
```

Ruff (0.16.0)

[error] Ambiguous variable name: `I`

(E741)